data/sitesEN.json

```diff
@@ -5135,6 +5135,28 @@
     "tags": [
       "official"
     ]
+  },
+    {
+    "id": "en-scpsl",
+    "origins_label": "SCP: Secret Laboratory Fandom Wiki",
+    "origins": [
+      {
+        "origin": "SCP: Secret Laboratory Fandom Wiki",
+        "origin_base_url": "scp-secret-laboratory-official.fandom.com",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "SCP:_Secret_Laboratory_Official_Wiki"
+      }
+    ],
+    "destination": "SCP: Secret Laboratory Wiki",
+    "destination_base_url": "en.scpslgame.com",
+    "destination_platform": "mediawiki",
+    "destination_icon": "scpslwiki.png",
+    "destination_main_page": "Main_Page",
+    "destination_search_path": "/index.php",
+    "destination_content_path": "/index.php?title="
```

Comment:
Missing trailing comma; please put the JSON through a validator to catch errors like these